opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -23,35 +23,37 @@ option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.trace.v1";
 option java_outer_classname = "TraceProto";
 
-// A span represents a single operation within a trace. Spans can be
+// Span represents a single operation within a trace. Spans can be
 // nested to form a trace tree. Spans may also be linked to other spans
-// from the same or different trace. And form graphs. Often, a trace
+// from the same or different trace and form graphs. Often, a trace
 // contains a root span that describes the end-to-end latency, and one
 // or more subspans for its sub-operations. A trace can also contain
 // multiple root spans, or none at all. Spans do not need to be
 // contiguous - there may be gaps or overlaps between spans in a trace.
 //
-// The next id is 15.
+// The next field id is 17.
 message Span {
-  // A unique identifier for a trace. All spans from the same trace share
-  // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
-  // is considered invalid.
+  // trace_id is the unique identifier of a trace. All spans from the same trace share
+  // the same `trace_id`. The ID is a 128 bit (16 byte) value represented using two
+  // fixed64 fields. An ID with value of 0 is considered invalid.
   //
-  // This field is semantically required. Receiver should generate new
-  // random trace_id if empty or invalid trace_id was received.
+  // This field is semantically required. If 0 or invalid trace_id was received:
+  // - The receiver MAY reject the invalid data and respond with the appropriate error
+  //   code to the sender.
+  // - The receiver MAY accept the invalid data and attempt to correct it.
+  // Low 64 bits of ID.
+  fixed64 trace_id_low = 1;
+  // High 64 bits of ID.
+  fixed64 trace_id_high = 2;
+
+  // span_id is a unique identifier for a span within a trace, assigned when the span
+  // is created. The ID is a 64 bit (8 byte) value. An ID with value of 0 is considered invalid.
   //
-  // This field is required.
-  bytes trace_id = 1;
-
-  // A unique identifier for a span within a trace, assigned when the span
-  // is created. The ID is an 8-byte array. An ID with all zeroes is considered
-  // invalid.
-  //
-  // This field is semantically required. Receiver should generate new
-  // random span_id if empty or invalid span_id was received.
-  //
-  // This field is required.
-  bytes span_id = 2;
+  // This field is semantically required. If 0 or invalid span_id was received:
+  // - The receiver MAY reject the invalid data and respond with the appropriate error
+  //   code to the sender.
+  // - The receiver MAY accept the invalid data and attempt to correct it.
+  fixed64 span_id = 3;
```

Comment:
Same as trace id. This is defined by the w3c standard that we use as a 8 bytes array. Not clear how that gets converted to a fixed64.